### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by zhaoxvwahaha** [lbcb-sci/racon#42]
Hi @rvaser ,

I used racon (v1.4.13) to polish assembly,there are 20+ samples, only 2 of them have the following  different problems:
(1) 
racon -m 8 -x -6 -g -8 -w 500 -t 8 sample1.porechopQC.filtered_reads.fq.gz sample1.sam sample1.flye.assembly.fasta > sample1.racon.fasta  
[racon::Polisher::initialize] loaded target sequences 0.225069 s
[racon::Polisher::initialize] loaded sequences 70.247542 s
[racon::Polisher::initialize] loaded overlaps 18.639637 s
[racon::Polisher::initialize] aligning overlaps [====================] 11.493455 s
Segmentation fault (core dumped)
(2)
racon -m 8 -x -6 -g -8 -w 500 -t 8 sample2.porechopQC.filtered_reads.fq.gz sample2.sam sample2.flye.assembly.fasta > sample2.racon.fasta
[racon::Polisher::initialize] loaded target sequences 0.068931 s
[racon::Polisher::initialize] loaded sequences 64.135422 s
[racon::Polisher::initialize] loaded overlaps 16.719191 s
[racon::Overlap::find_breaking_points] error: overlap is not transmuted!

Could you give some advice, Thanks in advance!
Hailong

**Comment by rvaser** [comment]
Hi Hailong,
can you paste command which produced `sample1.sam` and `sample2.sam`?

Best regards,
Robert

**Comment by zhaoxvwahaha** [comment]
bwa index 02.flye/Kp819/assembly.fasta
bwa mem 02.flye/Kp819/assembly.fasta 01.porechop_QC/Kp819/Kp819.porechopQC.filtered_reads.fq.gz >03.flye.asmcvg.racon/Kp819/Kp819.sam

**Comment by rvaser** [comment]
And what is the corresponding racon command? The errors you got indicate that you might misplaced input parameters.

**Comment by zhaoxvwahaha** [comment]
Hi @rvaser ,the following are racon commands:
(1)
racon -m 8 -x -6 -g -8 -w 500 -t 8 01.porechop_QC/Kp819/Kp819.porechopQC.filtered_reads.fq.gz 03.flye.asmcvg.racon/Kp819/Kp819.sam 02.flye.asmcvg/Kp819/assembly.fasta > 03.flye.asmcvg.racon/Kp819/Kp819.racon.fasta
(2)
racon -m 8 -x -6 -g -8 -w 500 -t 8 01.porechop_QC/Kp95/Kp95.porechopQC.filtered_reads.fq.gz 03.flye.asmcvg.racon/Kp95/Kp95.sam 02.flye.asmcvg/Kp95/assembly.fasta > 03.flye.asmcvg.racon/Kp95/Kp95.racon.fasta

**Comment by rvaser** [comment]
The commands look fine, if the `03.flye.asmcvg.racon/Kp95/Kp95.sam` file was generated with `02.flye.asmcvg/Kp95/assembly.fasta` and `01.porechop_QC/Kp95/Kp95.porechopQC.filtered_reads.fq.gz`. Is it possible for you to share those two assemblies and corresponding reads, so I can see what is happening?

**Comment by zhaoxvwahaha** [comment]
@rvaser 
yes, but how could I share the data to you, the data size is big (~2.5G)

**Comment by rvaser** [comment]
Dropbox? Google drive? Not sure :/
